VTEX - Antifraud Provider API.json

```diff
@@ -235,178 +88,33 @@
         "/transactions": {
             "post": {
                 "tags": [
-                    "Antifraud  Flow"
+                    "Anti-fraud Flow"
                 ],
-                "summary": "Send Antifraud Data",
-                "description": "Receives a new transaction antifraud data. This step is performed only if all payments are authorized.\r\n\r\nVTEX expects that your antifraud provider process this request and retrieves the process result. After that, the VTEX order flow will check the response status to cancel or approve the transaction.",
+                "summary": "Send Anti-fraud Data",
+                "description": "Receives a new transaction anti-fraud data. This step is performed only if all payments are authorized.\r\n\r\nVTEX expects that your anti-fraud provider process this request and retrieves the process result. After that, the VTEX order flow will check the response status to cancel or approve the transaction.\r\n\r\n## Permissions\r\n\r\nThis endpoint does not require [authentication](https:\/\/developers.vtex.com\/docs\/guides\/authentication) or [permissions](https:\/\/help.vtex.com\/en\/tutorial\/license-manager-resources--3q6ztrC8YynQf6rdc6euk3).",
```

Comment:
Se esse endpoint não requer authentication, por que tem security nele?

Reply:
Adicionado nova descrição, conforme conversamos: ## Permissions\r\n\r\nThis endpoint does not require [permissions](https:\/\/help.vtex.com\/en\/tutorial\/license-manager-resources--3q6ztrC8YynQf6rdc6euk3).